digdag-docs/src/operators/emr.md

```diff
@@ -41,6 +41,14 @@ For detailed information about EMR, see the [Amazon Elastic MapReduce Documentat
 
   The AWS Role to assume when submitting EMR jobs.
 
+* **aws.emr.region, aws.region**
+
+  The AWS region to use for EMR service.
+
+* **aws.emr.endpoint**
+
+  The AWS EMR [endpoint address](http://docs.aws.amazon.com/general/latest/gr/rande.html) to use.
+
```

Comment:
Could you add explanations about `aws.kms.*` and `aws.s3.*` here since they get available with this pull request?